crates/server/src/server/metrics.rs

```diff
@@ -0,0 +1,273 @@
+use hickory_proto::op::{Header, OpCode, ResponseCode};
+use hickory_proto::xfer::Protocol;
+use metrics::{Counter, Unit, counter, describe_counter};
+
+#[derive(Clone)]
+pub(super) struct ReportingResponseHandlerMetrics {
+    pub(super) request_proto: ProtocolMetrics,
```

Comment:
Let's rename `request_proto` to just `proto` and `request_operation` to just `operation`?

Reply:
Renamed.